{{cookiecutter.repostory_name}}/tools/backup-b2/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG BASE_IMAGE=python:3.8-alpine
+
+FROM $BASE_IMAGE AS base-image
+
+WORKDIR /root/
+
+RUN pip3 install --no-cache-dir -U pip
+RUN pip3 install --no-cache-dir b2
+
+COPY ./send_backup.sh /root/
```

Comment:
@mlech-reef I think those will create a new docker layer for every script we copy in, also whenever we add a script to this directory we'll need to add it here as well. Can't we use some wildcard thing with `.dockerignore` instead?

Reply:
Or just move to `/tools/backup-b2/src/`. Someone may forgot to update `.dockerignore` as it's hidden

Reply:
@mlech-reef  @ppolewicz I'm not clear on what I should change here after all - can you explain?

Reply:
The goal is to have only one `COPY` statement here. Either copy the entire directory/subdirectory or use `.dockerignore` and wildcards

Reply:
Used *.sh wildcard. .dockerignore not needed here